.github/workflows/close.yaml

```diff
@@ -0,0 +1,34 @@
+# This workflow closes issues that have had no activity for 90 days.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Close Old
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 1 1,15 * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-issue-stale: 30
+        close-issue-message: >
+          @author, We are closing this as there was no activity in this issue for last 90 days. Please reopen if you’d like to discuss anything further.
+        only-labels: 'obsolete'
+        exempt-issue-labels: awaiting-maintainer
+        ascending: true
+        days-before-close: 0
+        close-issue-reason: "not_planned"
+        stale-issue-label: 'closed'
```

Comment:
Can we add `wontfix` instead (https://github.com/googleforgames/agones/labels/wontfix) on close, since that is already ignored by the changelist, and aligns with what we already have?

Reply:
Done